src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -561,6 +561,21 @@ class Env_test : public beast::unit_test::suite
                 );
     }
 
+    void testSig()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        env.fund(XRP(10000), "alice");
+        auto const baseFee = env.app().config().FEE_DEFAULT;
+        std::uint32_t const aliceSeq = env.seq ("alice");
+        
+        Json::Value jsonNoop = env.json (
+            noop ("alice"), fee(baseFee), seq(aliceSeq), sig("alice"));
+        JTx jt = env.jt (jsonNoop);
+        env (jt);
```

Comment:
So there's a problem with either the change or the test:
1. `jsonNoop` is a `Json` object that includes a signature.
2. That `Json` object is passed to `env.jt()`, which generates a new `JTx`, which is resigned, ignoring the original signature from `jsonNoop`.
3. That `JTx` is submitted to `env`, which generates _yet another_ `JTx`, which is signed yet again, ignoring both original signatures.

In other words, the test will pass, no matter who signs `jsonNoop`. I added some more test cases at https://github.com/ximinez/rippled/commit/f11f00c85d6261365d5a519d32dace8c50d8f236, which demonstrate what I'm talking about. (They all pass.)

If that's intentional, great. Maybe cherry-pick those test cases so it's clearer to future readers. If not...